src/mapping/lending-pool-configurator/v3.ts

```diff
@@ -284,6 +286,50 @@ export function handleEModeCategoryAdded(event: EModeCategoryAdded): void {
   eModeCategory.save();
 }
 
+export function handleAssetCollateralInEModeChanged(event: AssetCollateralInEModeChanged): void {
+  let id = BigInt.fromI32(event.params.categoryId).toString();
+  let eModeCategory = EModeCategory.load(id);
+  if (!eModeCategory) {
+    log.error('Emode category with id: {} does not exist', [id]);
```

Comment:
I did a quick search of the code and I think it's safe to assume that the `EModeCategory` will always exist, but it would be good to do a more thorough check to see if there's a case where this event is triggered before the [`EModeCategoryAdded` event](https://github.com/aave-dao/aave-v3-origin/blob/ec33f4fcc5546710c0faca5208af1775c761ac9c/src/contracts/protocol/pool/PoolConfigurator.sol#L406)

Reply:
Updated so a placeholder EMode category entity will be created in the case where it does not exist yet